app/src/main/java/org/cru/godtools/ui/dashboard/tools/CategoryAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package org.cru.godtools.ui.dashboard.tools
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.ObservableField
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.Observer
+import org.ccci.gto.android.common.recyclerview.adapter.SimpleDataBindingAdapter
+import org.cru.godtools.databinding.DashboardListItemCategoryBinding
+
+class CategoryAdapter(lifecycleOwner: LifecycleOwner, private val dataModel: ToolsCategoryDataModel) :
+    SimpleDataBindingAdapter<DashboardListItemCategoryBinding>(lifecycleOwner), Observer<List<String>> {
+
+    val callbacks = ObservableField<CategoryAdapterCallbacks>()
+
+    private var categories: List<String> = emptyList()
+        set(value) {
+            field = value
+            notifyDataSetChanged()
+        }
+
+    override fun getItemCount() = categories.size
+
+    override fun onChanged(t: List<String>) {
+        categories = t
+    }
+
+    override fun onCreateViewDataBinding(parent: ViewGroup, viewType: Int) = DashboardListItemCategoryBinding
+        .inflate(LayoutInflater.from(parent.context), parent, false).also {
+            it.callbacks = callbacks
+        }
+
+    override fun onBindViewDataBinding(binding: DashboardListItemCategoryBinding, position: Int) {
+        val category = categories[position]
+        binding.category = category
+        binding.formattedCategory = dataModel.getFormattedCategory(category)
```

Comment:
you shouldn't need to set this on the binding, you should be able to directly call the ModelUtil method from the binding itself using the `category` name that you already have there.

The whole point is separation of concerns, when looking at the layout XML I want to be able to quickly tell how we are generating the category display name and not have to hunt where the name is being set to a variable